hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/TopicWithSchemaEndpoint.java

```diff
@@ -0,0 +1,42 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import pl.allegro.tech.hermes.api.PatchData;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.api.TopicWithSchema;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("topics-with-schema")
```

Comment:
i would rather have `TopicEndpoint` working with `TopicWithSchema` object in management, adding new endpoint feels weird